models/_sector/referral/rewards/platforms/rabbithole_arbitrum_rewards.sql

```diff
@@ -0,0 +1,17 @@
+{{ config(
+    schema = 'rabbithole_arbitrum',
+    alias = alias('rewards'),
+    tags = ['dunesql'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['project','tx_hash','sub_tx_id'],
+    incremental_predicates = ['DBT_INTERNAL_DEST.block_time >= date_trunc(\'day\', now() - interval \'7\' day)']
```

Comment:
these can now simplify to:
`incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],`

PR which implemented:
https://github.com/duneanalytics/spellbook/pull/4487/files